packages/embark-test-runner/src/index.js

```diff
@@ -91,6 +88,18 @@ class TestRunner {
     });
   }
 
+  generateCoverageReport() {
+    const coveragePath = path.join(dappPath(), ".embark", "coverage.json");
+    const coverageMap = JSON.parse(fs.readFileSync(coveragePath));
```

Comment:
This lien has a double potential to throw, should we wrap it in a try/catch or maybe the function call of `generateCoverageReport` should be?